bot/cogs/snakes.py

```diff
@@ -28,9 +90,60 @@ def __init__(self, bot: AutoShardedBot):
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         :return: A dict containing information on a snake
         """
+        global snakelist
+        snake_name = name
+        name = name.replace(" ", "_")  # sanitize name
+
+        text_params = {'action': 'query',
+                       'titles': name,
+                       'prop': 'extracts',
+                       'exsentences': '2',
+                       'explaintext': '1',
+                       'autosuggest': '1',
+                       'redirects': '1',
+                       'format': 'json'}
+
+        image_name_params = {'action': 'query',
+                             'titles': name,
+                             'prop': 'images',
+                             'redirects': '1',
+                             'autosuggest': '1',
+                             'imlimit': '1',
+                             'format': 'json'}
+
+        text_json = await self.get_wiki_json(text_params)
+        image_name_json = await self.get_wiki_json(image_name_params)
+        snake_image = "https://pbs.twimg.com/profile_images/662615956670144512/dqsVK6Nw_400x400.jpg"
+
+        page_id = list(text_json['query']['pages'].keys())[0]
+        if page_id == "-1" or snake_name not in snakelist:  # No entry on the wiki
+            snake_dict = {"name": snake_name,
+                          "snake_text": "You call that a snake?\nTHIS is a snake!",
+                          "snake_image": snake_image}
+            return snake_dict
+
+        image_id = image_name_json['query']['pages'][page_id]['images'][0]['title']
+
+        image_url_params = {'action': 'query',
+                            'titles': image_id,
+                            'prop': 'imageinfo',
+                            'redirects': '1',
+                            'autosuggest': '1',
+                            'iiprop': 'url',
+                            'format': 'json'}
+
+        image_url_json = await self.get_wiki_json(image_url_params)
+
+        snake_image_id = list(image_url_json['query']['pages'].keys())[0]
+        snake_image = image_url_json['query']['pages'][snake_image_id]['imageinfo'][0]['url']
+        snake_text = text_json['query']['pages'][page_id]['extract']
+
+        snake_dict = {"name": snake_name, "snake_text": snake_text, "snake_image": snake_image}
+        return snake_dict
 
     @command()
     async def get(self, ctx: Context, name: str = None):
+        global snakelist
```

Comment:
This code is invalid as Python docstrings will only work when on the first line of a function, take a look at this example:
```py
>>> def func():
...     """Hello this is my docstring"""
...     print("Hello")
... 
>>> func.__doc__
'Hello this is my docstring'
>>> def func():
...     print("Hello")
...     """Hello this is my docstring"""
... 
>>> func.__doc__
None
>>> 
```

Reply:
Runew0lf wrote it.

Reply:
Runew0lf fixed it